Remotion/Data/DomainObjects/Tracing/TracingDbTransaction.cs

```diff
@@ -61,39 +62,35 @@ public IPersistenceExtension PersistenceExtension
       get { return _persistenceExtension; }
     }
 
-    public void Dispose ()
+    protected override void Dispose (bool disposing)
     {
-      _transaction.Dispose();
-
-      if (!_isTransactionDisposed)
+      if (disposing)
       {
-        PersistenceExtension.TransactionDisposed(_connectionID);
-        _isTransactionDisposed = true;
+        _transaction.Dispose();
+
+        if (!_isTransactionDisposed)
+        {
+          PersistenceExtension.TransactionDisposed(_connectionID);
+          _isTransactionDisposed = true;
+        }
       }
     }
 
-    public void Commit ()
+    public override void Commit ()
     {
       _transaction.Commit();
       if (!_isTransactionDisposed)
         PersistenceExtension.TransactionCommitted(_connectionID);
     }
 
-    public void Rollback ()
+    public override void Rollback ()
     {
       _transaction.Rollback();
       if (!_isTransactionDisposed)
         PersistenceExtension.TransactionRolledBack(_connectionID);
     }
 
-    IDbConnection? IDbTransaction.Connection
-    {
-      get { return _transaction.Connection; }
-    }
-
-    public IsolationLevel IsolationLevel
-    {
-      get { return _transaction.IsolationLevel; }
-    }
+    protected override DbConnection? DbConnection => _transaction.Connection;
+    public override IsolationLevel IsolationLevel => _transaction.IsolationLevel;
```

Comment:
```suggestion
    protected override DbConnection? DbConnection => _transaction.Connection;

    public override IsolationLevel IsolationLevel => _transaction.IsolationLevel;
```

Am considering, move them up to the other properties before the methods.